layers/datadog/Dockerfile

```diff
@@ -7,15 +7,15 @@ ENV DDTRACE_BUILD_DIR=${BUILD_DIR}/ddtrace
 RUN set -xe; \
     mkdir -p ${DDTRACE_BUILD_DIR}; \
     curl -Ls -o ${DDTRACE_BUILD_DIR}/datadog-setup.php \
-      https://github.com/DataDog/dd-trace-php/releases/download/0.96.0/datadog-setup.php
+      https://github.com/DataDog/dd-trace-php/releases/download/1.5.1/datadog-setup.php
 
 WORKDIR ${DDTRACE_BUILD_DIR}
 
 RUN php datadog-setup.php --php-bin=all --enable-profiling
 
 RUN cp "$(php-config --extension-dir)/ddtrace.so" /tmp/ddtrace.so && \
     cp "$(php-config --extension-dir)/datadog-profiling.so" /tmp/datadog-profiling.so && \
-    cp "$(php-config --ini-dir)/98-ddtrace.ini" /tmp/ext.ini
+    cp "$(php-config --ini-dir | cut -d: -f1)/98-ddtrace.ini" /tmp/ext.ini
```

Comment:
Could you explain this change?

Reply:
The `php-config --ini-dir` returns `/opt/bref/etc/php/conf.d:/var/task/php/conf.d`. This `cp` command assumes that there is only single path in the output, otherwise the source file is not found. Probably, there was some change in build-php image recently, which changed the behavior and now contains two paths separated by colon. If you have better idea for fix, I would be happy to test and implement it.